docs/AGGREGATORS.md

```diff
@@ -22,6 +22,8 @@ This section is for developers who want to create a new aggregator plugin.
 ### Aggregator Plugin Example
 
 ```go
+//go:generate go run ../../../tools/generate_plugindata/main.go
+//go:generate go run ../../../tools/generate_plugindata/main.go --clean
```

Comment:
For users who import our modules in an external go module, I assume unless they run the `go generate` command itself they will only get `{{ .SampleConfig }}` from the SampleConfig function?